reflekt-plugin/src/main/kotlin/io/reflekt/plugin/analysis/processor/Util.kt

```diff
@@ -12,3 +12,7 @@ internal val KtElement.isPublicFunction: Boolean
 
 internal val KtElement.isPublicNotAbstractClass: Boolean
     get() = this is KtClass && this.isPublic && !this.isAbstract()
+
+// TODO: is it enough to identify a file?
+internal val KtFile.fullName: String
```

Comment:
Maybe it’s better to create a typealias for FileID and use it everywhere instead of just String? Thus it would be clearer that we store files and it would be simpler to change it everywhere on a different type in case fullName is not enough

Reply:
great idea, thank you!